src/DotNetCore.CAP.PostgreSql/CAP.PostgreSqlOptions.cs

```diff
@@ -16,6 +17,19 @@ public class PostgreSqlOptions : EFOptions
     /// Gets or sets the database's connection string that will be used to store database entities.
     /// </summary>
     public string ConnectionString { get; set; } = default!;
```

Comment:
Just some food for thought here: I know that we're keeping this for backward compatibility purposes, but would it make sense to mark it as obsolete and indicate that we intend to replace it with the `NpgsqlDataSource` in future versions?

Reply:
I think it looks great. @yang-xiaodong what's your opinion over this deprecation notice for this property?!

Reply:
Our next release 8.1.0 is a minor version, So I think `[Obsolete]` it is acceptable.

Reply:
Ok, I have marked ConnectionString obsolete.